### DISCUSSION TITLE
How can I hide the line of the Konva rotation handle and keep the rotation function?

### DISCUSSION THREAD

**Discussion by liulinboyi** [konvajs/konva#1688] [Q&A] [answered]
How can I hide the line of the Konva rotation handle and keep the rotation function? thank you!

like the follow:

![image](https://github.com/konvajs/konva/assets/41336612/be333b13-0ac0-4922-951f-0d3718686d4c)

![image](https://github.com/konvajs/konva/assets/41336612/5d873773-b457-4fd7-8e25-3d81639f60ae)

**Comment by lavrton** [comment]
`Konva.Transformer` doesn't have API for that. Do you want to make a Pull Request?

**Reply by liulinboyi** [reply]
> `Konva.Transformer` doesn't have API for that. Do you want to make a Pull Request?

I will have a try, https://github.com/konvajs/konva/pull/1690

**Reply by lavrton** [reply, marked as answer]
Released your pull request https://github.com/konvajs/konva/pull/1690 with new `rotateLineVisible` attribute.

**Reply by liulinboyi** [reply]
> Released your pull request #1690 with new `rotateLineVisible` attribute.

Cool!
